docs/detections/rules-ui-create.asciidoc

```diff
@@ -560,10 +572,6 @@ used as an identifier across systems
 * `{{rule.type}}`: Type of rule
 * `{{state.signals_count}}`: Number of alerts detected
```

Comment:
It still feels like `{{state.signals_count}}` variable belongs only to `for each alert` case, but it is available and more valuable for `summary of alerts`.

Reply:
@e40pud would you mind taking a look at the [Alert notification placeholders](https://security-docs_3245.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#rule-action-variables) section to see if the intro para correctly conveys which variables are passed for the frequency options?

Reply:
Yes, it looks correct. Though, I think the `{{context}} variables` for summary alerts is not enough.

I think it would be better to tell that all these variables are available for both `summary` and `per alert`:

```
{{context.alerts}}: Array of detected alerts
{{{context.results_link}}}: URL to the alerts in Kibana
{{context.rule.anomaly_threshold}}: Anomaly threshold score above which alerts are generated (machine learning rules only)
{{context.rule.description}}: Rule description
{{context.rule.false_positives}}: Rule false positives
{{context.rule.filters}}: Rule filters (query rules only)
{{context.rule.id}}: Unique rule ID returned after creating the rule
{{context.rule.index}}: Indices rule runs on (query rules only)
{{context.rule.language}}: Rule query language (query rules only)
{{context.rule.machine_learning_job_id}}: ID of associated machine learning job (machine learning rules only)
{{context.rule.max_signals}}: Maximum allowed number of alerts per rule execution
{{context.rule.name}}: Rule name
{{context.rule.query}}: Rule query (query rules only)
{{context.rule.references}}: Rule references
{{context.rule.risk_score}}: Default rule risk score

The {{context.rule.risk_score}} placeholder contain the rule’s default values even when the Risk score override option is used.
{{context.rule.rule_id}}: Generated or user-defined rule ID that can be used as an identifier across systems
{{context.rule.saved_id}}: Saved search ID
{{context.rule.severity}}: Default rule severity

The {{context.rule.severity}} placeholder contain the rule’s default values even when the Severity override option is used.
{{context.rule.threat}}: Rule threat framework
{{context.rule.threshold}}: Rule threshold values (threshold rules only)
{{context.rule.timeline_id}}: Associated Timeline ID
{{context.rule.timeline_title}}: Associated Timeline name
{{context.rule.type}}: Rule type
{{context.rule.version}}: Rule version
{{date}}`: Date the rule scheduled the action
{{kibanaBaseUrl}}: Configured server.publicBaseUrl value, or empty string if not configured
{{rule.id}}: ID of the rule
{{rule.name}}: Name of the rule
{{rule.spaceId}}: Space ID of the rule
{{rule.tags}}: Tags of the rule
{{rule.type}}: Type of rule
{{state.signals_count}}: Number of alerts detected
```

and that the next variables are available only for `per alert`:

```
{{alert.actionGroup}}: Action group of the alert that scheduled actions for the rule
{{alert.actionGroupName}}: Human readable name of the action group of the alert that scheduled actions for the rule
{{alert.actionSubgroup}}: Action subgroup of the alert that scheduled actions for the rule
{{alert.id}}: ID of the alert that scheduled actions for the rule
{{alert.flapping}}`: A flag on the alert that indicates whether the alert status is changing repeatedly
```

Reply:
Addressed in [639c5a5](https://github.com/elastic/security-docs/pull/3245/commits/639c5a5a2a726cfc0b8338c7ad3f54c76df8a2b9).